controllers/profiles/reconciler_dev.go

```diff
@@ -324,11 +358,108 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for keycm, cm := range externalResourceConfigMaps {
+				deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+					v1.Volume{
+						Name: cm.Name,
+						VolumeSource: v1.VolumeSource{
+							ConfigMap: &v1.ConfigMapVolumeSource{
+								LocalObjectReference: v1.LocalObjectReference{Name: cm.Name},
```

Comment:
This is exactly the same code for the `flowDefCM`. I believe we can create all of them in one go.

Reply:
Do you mean one function to use for every volume and volumeMount creation ?

Reply:
You can figure out a way to mount all the cms in one cycle to avoid code duplication.

Reply:
I'll wite first a compact way then I'll try to figure out how to add the externals cms without iterate

Reply:
@ricardozanini done